### PR TITLE
fix: Android Import/Export escaping

### PR DESCRIPTION
**Pull request by JanCizmar**

### CODE REVIEW

File: backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package io.tolgee.formats.android
+
+const val ANDROID_CDATA_CUSTOM_KEY = "_androidWrapWithCdata"
```
Comment (stepan662) [inline backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt:3]: I don't like the underscosre

**Review by stepan662 — APPROVED** [review]

File: backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt (hunk 1 above)
Comment (JanCizmar) [reply backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt:3]: What don't you like about it?

Comment (stepan662) [reply backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt:3]: It's in section described as "Custom attributes specific for some formats", so it's basically the section exactly for this. The "_" looks hacky.

Snímek obrazovky 2024-03-19 v 15 16 34

Comment (JanCizmar) [reply backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt:3]: https://tolgee.io/platform/translation_keys/custom_values#tolgee-internal-custom-values

**Pull request merged**